packages/api/src/external/fhir/document/index.ts

```diff
@@ -378,7 +378,7 @@ export function containedPractitionerToFHIRResource(
     id: resource.id ?? undefined,
     identifier: convertCWIdentifierToFHIR(resource.identifier),
     name: convertCWNameToHumanName(resource.name),
-    gender: convertCWGenderToFHIR(resource.gender?.coding),
+    gender: convertCWGenderToFHIR(resource.gender?.coding as unknown as Gender[]),
```

Comment:
had to do these b/c was getting a compilation error I couldn't fix after a few attempts, it was becoming a rabbit hole.